code/modules/games/cards/lotto.dm

```diff
@@ -0,0 +1,130 @@
+/obj/item/toy/lotto_ticket
+	name = "scratch-off lotto ticket"
+	desc = "A scratch-off lotto ticket."
+	icon = 'icons/obj/toy.dmi'
+	w_class = W_CLASS_TINY
+	var/obj/item/toy/lotto_ticket/revealed = 0
+	var/obj/item/toy/lotto_ticket/iswinner = 0
+	var/obj/item/toy/lotto_ticket/ticket_price
+	var/obj/item/toy/lotto_ticket/winnings = 0
```

Comment:
These vars are all defined with the type of `/obj/item/toy/lotto_ticket` but they all hold numbers (revealed and iswinner are only TRUE and FALSE, too).

Reply:
Resolved in latest commit